ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/mpls.cfg

```diff
@@ -24,3 +24,34 @@ mpls ldp
    no shutdown
 !
 mpls icmp fragmentation-needed tunneling
+!
+mpls rsvp
```

Comment:
can you please try to deploy the config via eapi role

Reply:
Fixed!

Reply:
Checked!